spec/features/merchants/items/index_spec.rb

```diff
@@ -92,6 +152,52 @@
         click_link "Create New Item"
         expect(current_path).to eq("/merchants/#{merchant_1.id}/items/new")
       end
+
+    #  User Story 12 - Merchant Items Index: 5 most popular items
+    #   As a merchant
+    #   When I visit my items index page
+    #   Then I see the names of the top 5 most popular items ranked by total revenue generated
+    #   And I see that each item name links to my merchant item show page for that item
+    #   And I see the total revenue generated next to each item name
+
+    #   Notes on Revenue Calculation:
+
+    #   Only invoices with at least one successful transaction should count towards revenue
+    #   Revenue for an invoice should be calculated as the sum of the revenue of all invoice items
+    #   Revenue for an invoice item should be calculated as the invoice item unit price multiplied
+    #   by the quantity (do not use the item unit price)
+
+      it "displays the top 5 most popular items ranked by total revenue for the one merchant" do
+        visit "/merchants/#{merchant_1.id}/items"
+        within ".top-five-items" do
+          expect(page).to have_content("Top 5 Items")
+
+          expect(page).to have_content("#{item_6.name} - $24,000.00 in sales")
+          expect(page).to have_content("#{item_1.name} - $18,000.00 in sales")
+          expect(page).to have_content("#{item_3.name} - $12,000.00 in sales")
+          expect(page).to have_content("#{item_2.name} - $6,000.00 in sales")
+          expect(page).to have_content("#{item_5.name} - $200.00 in sales")
+          expect(page).to_not have_content("#{item_4.name} - $24.00 in sales")
```

Comment:
In this testing here since we want the order to be most sales to least sales, we could test the order of them within ".top-five-items" using orderly. Like, `expect("#{item_6.name}").to appear_before("#{item_1.name}")`
I don't believe that just putting them in order in this test how you have it would actually test the order of them.